packages/js-client-rest/src/qdrant-client.ts

```diff
@@ -1429,8 +1434,13 @@ export class QdrantClient {
      *         Default: `replica`
      * @returns True if the snapshot was recovered
      */
-    async recoverSnapshot(collection_name: string, {location, priority}: SchemaFor<'SnapshotRecover'>) {
-        const response = await this._openApiClient.snapshots.recoverFromSnapshot({collection_name, location, priority});
+    async recoverSnapshot(collection_name: string, {location, priority, checksum}: SchemaFor<'SnapshotRecover'>) {
+        const response = await this._openApiClient.snapshots.recoverFromSnapshot({
+            collection_name,
+            location,
+            priority,
+            checksum: checksum === null ? undefined : checksum,
```

Comment:
I don't know how to avoid this check, `this._openApiClient.snapshots.recoverFromSnapshot` requires `checksum?: string | undefined` and `SchemaFor` get `checksum: string | null | undefined`

Reply:
Fixed, it was wrong openapi